gym/core.py

```diff
@@ -51,6 +51,7 @@ def __new__(cls, *args, **kwargs):
         env = super(Env, cls).__new__(cls)
         env._env_closer_id = env_closer.register(env)
         env._closed = False
+        env._spec = None
```

Comment:
Ah good point. I'd rather not change it to being a property on env too unless we have to. A bit of a hack but you could do: `if not hasattr(env, "spec"): env.spec = None`.